src/solvers/dgmulti/dg_parabolic.jl

```diff
@@ -195,9 +204,25 @@ function calc_viscous_fluxes!(viscous_flux, u, u_grad, mesh::DGMultiMesh,
   end
 end
 
+function calc_viscous_penalty!(scalar_flux_face_values, u_face_values, t, boundary_conditions,
+                               mesh, equations::AbstractEquationsParabolic, dg::DGMulti,
+                               dg_parabolic, cache, cache_parabolic)
+  # compute fluxes at interfaces
+  @unpack scalar_flux_face_values, inv_h = cache_parabolic
+  @unpack mapM, mapP = mesh.md
+  @threaded for face_node_index in each_face_node_global(mesh, dg)
+    idM, idP = mapM[face_node_index], mapP[face_node_index]
+    uM, uP = u_face_values[idM], u_face_values[idP]
+    inv_h_face = inv_h[face_node_index]
+    scalar_flux_face_values[idM] = scalar_flux_face_values[idM] + penalty(uP, uM, inv_h_face, equations, dg_parabolic)
```

Comment:
Where is `penalty` for `BR1` defined?

Reply:
It is not. `calc_viscous_penalty!` dispatches to doing nothing for `BR1()`

Reply:
Okay, thanks!